src/odemis/dataio/tiff.py

```diff
@@ -2696,7 +2696,7 @@ def _convert_thermo_fisher_to_odemis_metadata(metadata: str) -> dict:
     # parse ini metadata as dictionary
     tfs_md = {}
     try:
-        cfg = configparser.ConfigParser()
+        cfg = configparser.ConfigParser(strict=False) # allow duplicate keys
```

Comment:
Can you update the warning in line `2703` stating now that parsing duplicate keys was allowed?